app/models/mixins/relationship_mixin.rb

```diff
@@ -26,6 +27,7 @@ module RelationshipMixin
     extend Memoist
 
     cattr_accessor :default_relationship_type
+    class_attribute :skip_relationships, :default => []
```

Comment:
this looks great - although, not sure if we want this to carry across all the classes.  Will we tell all classes to skip ancestry that are in the chain?  If yes, then this is great.